test_integration/01_sequential_setup/02_organization_helper.js

```diff
@@ -48,52 +51,29 @@ describe('OrganizationHelper', () => {
   });
 
   after(() => {
-    const auxiliaryHelper = new OrganizationHelper(shared.auxiliary.web3);
-    return auxiliaryHelper.setup(orgConfig).then(() => {
-      shared.auxiliary.addresses.Organization = auxiliaryHelper.address;
-    });
+    return Organization.setup(shared.auxiliary.web3, orgConfig).then(
+      (instance) => {
+        shared.auxiliary.addresses.Organization = instance.address;
+      },
+    );
   });
 
   const subject = new OrganizationHelper(shared.origin.web3);
 
   it('should deploy new organization contract', () => {
     return subject
```

Comment:
Could we rename all instances of `subject` with the name of the contract? I always need to go back and check what the subject is right now.

Reply:
Create follow up ticket.

Reply:
#100